docker/configs/prometheus.yml

```diff
@@ -0,0 +1,42 @@
+    scrape_configs:
+      - job_name: 'mqtt-metrics'
+        basic_auth:
+          username: 'prometheus'
+          password: "$PROMETHEUS_PASSWORD"
+        scheme: 'http'
+        static_configs:
+          - targets:
+            - 'mosquitto:$PORT_MOSQUITTO_MQTT'
+        relabel_configs:
+          - source_labels: [__address__]
+            target_label: instance
+            regex: '([^:]+)(:[0-9]+)?'
+            replacement: '${1}'
+
+#      - job_name: 'kibana'
+#        basic_auth:
+#          username: 'readall'
+#          password: 'any-password'
+#        scheme: 'https'
+#        metrics_path: /_prometheus/metrics
+#        static_configs:
+#          - targets:
+#            - 'kibana:${PORT_KIBANA}'
+#        tls_config:
+#          insecure_skip_verify: true
```

Comment:
Should Kibana scraping be enabled? Could be a nice addition, but is not mandatory for now.